evalml/automl/automl_algorithm/iterative_algorithm.py

```diff
@@ -129,6 +193,88 @@ def __init__(
                         " and Real!"
                     )
 
+    def _create_pipelines(self):
```

Comment:
This logic is ripped out of `AutoMLSearch` and the API changes in `IterativeAlgorithm` accommodates this. Now `DefaultAlgorithm` and `IterativeAlgorithm` have more similar APIs.

Reply:
I meant the `IterativeAlgorithm.__init__()` parameters!